api/src/main/resources/dao/pipeline-run-dao.xml

```diff
@@ -1358,6 +1358,62 @@
                 ]]>
             </value>
         </property>
+        <property name="loadAllRunsPossiblyActiveInPeriodQuery">
+            <value>
+                <![CDATA[
+                    SELECT
+                        run_id,
+                        pipeline_id,
+                        version,
+                        start_date,
+                        end_date,
+                        parameters,
+                        status,
+                        terminating,
+                        pod_id,
+                        node_type,
+                        node_disk,
+                        node_ip,
+                        node_name,
+                        node_id,
+                        node_image,
+                        node_cloud_region,
+                        docker_image,
+                        cmd_template,
+                        actual_cmd,
+                        timeout,
+                        owner,
+                        service_url,
+                        pod_ip,
+                        commit_status,
+                        last_change_commit_time,
+                        config_name,
+                        node_count,
+                        parent_id,
+                        entities_ids,
+                        is_spot,
+                        configuration_id,
+                        pod_status,
+                        prolonged_at_time,
+                        last_notification_time,
+                        last_idle_notification_time,
+                        exec_preferences,
+                        pretty_url,
+                        price_per_hour,
+                        state_reason,
+                        non_pause,
+                        node_real_disk,
+                        node_cloud_provider,
+                        tags
+                    FROM
+                        pipeline.pipeline_run r
+                    WHERE
+                        r.status = 2
```

Comment:
I think condition should be the following: latest status from run_statuses (filtered by date) shall be not final

Reply:
Done